src/lib/macos-compat.h

```diff
@@ -0,0 +1,24 @@
+#if !defined(NOTCURSES_MACOS_COMPAT_H)
+#define NOTCURSES_MACOS_COMPAT_H
+
+#include <poll.h>
+
+#if !defined(TIMER_ABSTIME)
+// Value doesn't matter, mac builds don't use it anyway
+#define TIMER_ABSTIME 0
```

Comment:
so `TIMER_ABSTIME` is used in conjunction with `clock_nanosleep` in at least:

 * fades (lib/fade.c)
 * streaming (lib/visual.cpp)
 * demo control flow (all over the place)